packages/desktop/views/dashboard/collectibles/views/DetailsView.svelte

```diff
@@ -10,48 +10,19 @@
         Text,
     } from 'shared/components'
     import { localize } from '@core/i18n'
+    import { selectedNftId } from '../stores/selected-nft.store'
+    import { getNftByIdFromAllAccountNfts, INft } from '@core/nfts'
+    import { selectedAccountIndex } from '@core/account'
+    import { truncateString } from '@core/utils'
 
-    export let id: string = 'vt7ryeasd6f5hfh689w'
-    export let name: string = 'IOTABOTS #5'
-    export let collection: string = 'IOTABOTS'
-    export let collectionUrl: string = 'https://iotabots.io'
-    export let type: string = 'IRC27'
-    export let storageDeposit: string = '0.5 MI'
-    export let attributes: { title: string; value: string | number | boolean }[] = [
-        {
-            title: 'Background',
-            value: 'Purple',
-        },
-        {
-            title: 'Element',
-            value: 'Water',
-        },
-        {
-            title: 'Attack',
-            value: 150,
-        },
-        {
-            title: 'Super Long Attribute',
-            value: 2000,
-        },
-        {
-            title: 'Another Attribute',
-            value: 1000,
-        },
+    const nft: INft = getNftByIdFromAllAccountNfts($selectedAccountIndex, $selectedNftId)
 
-        {
-            title: 'Long Attribute',
-            value: 2000,
-        },
-        {
-            title: 'Another Attribute',
-            value: 1000,
-        },
-    ]
+    const { id, name } = nft
+    const { type, collectionName, attributes } = nft.parsedMetadata
 
     let modal: Modal
 
-    $: displayedId = id.substring(0, 6) + '...' + id.substring(id.length - 6, id.length)
+    const storageDeposit = 0
```

Comment:
Is this a const 0 until we develop this further? If so could you mark it with a comment?

Reply:
yes, it is for future work. Will add a comment :+1: